_sass/_layout.scss

```diff
@@ -573,27 +581,28 @@
         width: auto;
     }
 
-    ul, ol {
+    ol,
+    ul {
         margin-left: 0;
         padding-left: 15px;
     }
 }
-
 /* Language list */
-.en:lang(en), .es:lang(es), .pt:lang(pt), .fr:lang(fr) {
+.en:lang(en),
```

Comment:
Maybe via CSS variables?

Check browser support, but it could be an enhancement.